pkg/flow/config/processor_test.go

```diff
@@ -0,0 +1,107 @@
+/*
+* Flow CLI
```

Comment:
Should this file just be deleted?

Reply:
Nope, it should be uncommented and fixed - on it